lib/vector-buffers/src/variants/disk_v2/tests/mod.rs

```diff
@@ -273,8 +305,12 @@ where
     P: AsRef<Path>,
     R: Bufferable,
 {
+    let max_record_size =
+        usize::try_from(max_data_file_size).expect("Vector only supports 64-bit architectures.");
```

Comment:
Is this worth a wrapper function?

Reply:
I think in the tests, I'm just gonna keep the casts and switch to `unwrap`... since all a wrapper would end up looking like, I think, is something that resembles `max_data_file_size_to_max_record_size` which is just the same code -- `usize::try_from` -- and feels extraneous to me?

Reply:
Addressed in c54ae438f.